.github/workflows/gradle.yml

```diff
@@ -16,11 +16,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 11
+    - name: Set up JDK 17
```

Comment:
Do we need to migrate to Java 17 if we make this change?

Reply:
No, this PR is compatible with JDK 11 and 17 because we had set source compatibility to 1.8 in the Gradle build file.
This line asks GitHub Action to use the latest Java LTS version for code building, and we can keep using JDK 11 if we think it is better.
Thanks for pointing it out.

Reply:
I will keep the JDK version in 11. Thank you.